static/panes/gccdump-view.ts

```diff
@@ -397,7 +397,7 @@ export class GccDump extends MonacoPane<monaco.editor.IStandaloneCodeEditor, Gcc
             this.eventHub.emit(
                 'gccDumpFiltersChanged',
                 this.compilerInfo.compilerId,
-                this.getEffectiveFilters() as unknown as CompilerFilters,
+                this.getEffectiveFilters() as Record<string, boolean>,
```

Comment:
Is the cast even needed?

Reply:
Good question. In a first step I've tried to apply some renaming, then "correct the type". I don't see why we would need a cast here, so it could probably go away!

Reply:
I've played a bit more with types, looks like removing the cast is working, but I should test all features before saying that...